### PR TITLE
MCS-416 (Part2): Remove id, image_name from goal targets for metadata tiers

### PR DESCRIPTION
**Pull request by brianpippin**

### CODE REVIEW

File: scripts/create_videos.py [hunk 1]
```diff
@@ -20,10 +20,9 @@ def run():
     sample_number = sys.argv[3]
 
     folder_list = glob.glob(folder_prefix + '*')
-    folder_list = [
+    folder_list = sorted([
         folder for folder in folder_list if os.path.isdir(folder)
-    ]
-    folder_list.sort()
+    ])
```
Comment (brianpippin) [inline scripts/create_videos.py:25]: 'autopep3 --in-place --aggressive --recursive . 'changes

File: machine_common_sense/controller.py [hunk 2]
```diff
@@ -865,11 +865,14 @@ def restrict_step_output_metadata(self, step_output):
...
-                ):
-                    step_output.goal.metadata[target_name]['image'] = None
+                if (target_name in step_output.goal.metadata):
+                    if 'image' in step_output.goal.metadata[target_name]:
+                        step_output.goal.metadata[target_name]['image'] = None
+                    if 'id' in step_output.goal.metadata[target_name]:
+                        step_output.goal.metadata[target_name]['id'] = None
+                    if 'image_name' in step_output.goal.metadata[target_name]:
+                        step_output.goal.metadata[
+                            target_name]['image_name'] = None
```
Comment (brianpippin) [inline machine_common_sense/controller.py:875]: Flake8 complained about line too long without it being on two lines.

**Review by rartiss55 — APPROVED** [review]

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Review by deanwetherby — APPROVED** [review]

**Review by deanwetherby — COMMENTED** [review]
I am getting flake8 errors in test_controller

```(mcs) dwetherby@pc-2394-dw:~/workspace/MCS$ flake8
./tests/test_controller.py:1515:80: E501 line too long (80 > 79 characters)
./tests/test_controller.py:1516:80: E501 line too long (82 > 79 characters)
./tests/test_controller.py:1517:80: E501 line too long (81 > 79 characters)
./tests/test_controller.py:1577:80: E501 line too long (80 > 79 characters)
./tests/test_controller.py:1578:80: E501 line too long (82 > 79 characters)
./tests/test_controller.py:1579:80: E501 line too long (81 > 79 characters)```

**Comment by brianpippin** [comment]
> I am getting flake8 errors in test_controller
> 
> ```
> ./tests/test_controller.py:1515:80: E501 line too long (80 > 79 characters)
> ./tests/test_controller.py:1516:80: E501 line too long (82 > 79 characters)
> ./tests/test_controller.py:1517:80: E501 line too long (81 > 79 characters)
> ./tests/test_controller.py:1577:80: E501 line too long (80 > 79 characters)
> ./tests/test_controller.py:1578:80: E501 line too long (82 > 79 characters)
> ./tests/test_controller.py:1579:80: E501 line too long (81 > 79 characters)```
> ```

Fixed

**Pull request merged**